builder-support/dockerfiles/Dockerfile.dnsdist

```diff
@@ -12,7 +12,6 @@ RUN mkdir /sdist
 ARG BUILDER_VERSION
 
 RUN cd /dnsdist/pdns/dnsdistdist && \
-    meson setup build && \
-    meson dist -C build --no-tests
-WORKDIR /dnsdist/pdns/dnsdistdist
-RUN cp build/meson-dist/dnsdist-${BUILDER_VERSION}.tar.xz /sdist/
+    meson setup /tmp/dnsdist-meson-dist-build && \
```

Comment:
I normally don't like fixed names for /tmp dirs, but I suppose there's no chance of two build processes interfering here.